bytes/concat.ts

```diff
@@ -8,18 +8,41 @@
  * const a = new Uint8Array([0, 1, 2]);
  * const b = new Uint8Array([3, 4, 5]);
  * console.log(concat(a, b)); // [0, 1, 2, 3, 4, 5]
+ * ```
  */
-export function concat(...buf: Uint8Array[]): Uint8Array {
+export function concat(...buf: (Uint8Array | Uint8Array[])[]): Uint8Array {
```

Comment:
Could you overload these operators like so?
```suggestion
export function concat(buf: Uint8Array[]): Uint8Array;
export function concat(...buf: Uint8Array[]): Uint8Array;
export function concat(...buf: (Uint8Array | Uint8Array[])[]): Uint8Array {
```

Reply:
Sure, but in this case it won't work for multiple arrays of Uint8Array. So it will support only one array.

I will make a change as suggested.